app/views/my_facilities/drug_stocks/_drug_stocks_table.html.erb

```diff
@@ -3,9 +3,12 @@
     <colgroup>
       <col>
       <col>
-      <% @drugs_by_category.map do |_drug_category, drugs| %>
-        <col class="table-divider">
-        <% drugs.map do |_drug| %>
+      <% @drugs_by_category.map do |drug_category, drugs| %>
+        <% drugs.each_with_index.map do |_drug, index| %>
+          <col class="<%= "table-divider" if index.zero? %>">
+        <% end %>
+
+        <% unless drug_category == "diabetes" %>
```

Comment:
Let's extract this to a helper method like `show_patient_days?`.

Reply:
Good call, similar to the CSV exporter.

Reply:
Donezo.